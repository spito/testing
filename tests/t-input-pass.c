#include "cut.h"

#include <stdio.h>

TEST(input) {
    INPUT("6 5");
    int a;
    int b;

    ASSERT(2 == scanf("%d%d", &a, &b));
    ASSERT(a == 6);
    ASSERT(b == 5);
}