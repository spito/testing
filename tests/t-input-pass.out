[  1] input..................................................................OK

Summary:
  tests:          1
  succeeded:      1
  filtered out:   0
  suppressed:     0
  skipped:        0
  failed:         0
